velox/exec/HashTable.cpp

```diff
@@ -1291,10 +1291,12 @@ void HashTable<ignoreNullKeys>::decideHashMode(int32_t numNew) {
   if (bestWithReserve != VectorHasher::kRangeTooLarge) {
     enableRangeWhereCan(rangeSizes, distinctSizes, useRange);
     setHasherMode(hashers_, useRange, rangeSizes, distinctSizes);
+    setHashMode(HashMode::kNormalizedKey, numNew);
```

Comment:
can we move this out of if?

Reply:
Yes, you're right. Fixed in the latest commit.